### ISSUE TITLE
store extended user information

### ISSUE THREAD

**Issue by h1alexbel** [h1alexbel/fakehub#137]
we should have something like this:

```json
{
  "login": "h1alexbel",
  "id": 83453843,
  "node_id": "MDQ6VXNlcjgzNDUzODQz",
  "avatar_url": "https://avatars.githubusercontent.com/u/83453843?v=4",
  "gravatar_id": "",
  "url": "https://api.github.com/users/h1alexbel",
  "html_url": "https://github.com/h1alexbel",
  "followers_url": "https://api.github.com/users/h1alexbel/followers",
  "following_url": "[https://api.github.com/users/h1alexbel/following{/other_user}](https://api.github.com/users/h1alexbel/following%7B/other_user%7D)",
  "gists_url": "[https://api.github.com/users/h1alexbel/gists{/gist_id}](https://api.github.com/users/h1alexbel/gists%7B/gist_id%7D)",
  "starred_url": "[https://api.github.com/users/h1alexbel/starred{/owner}{/repo}](https://api.github.com/users/h1alexbel/starred%7B/owner%7D%7B/repo%7D)",
  "subscriptions_url": "https://api.github.com/users/h1alexbel/subscriptions",
  "organizations_url": "https://api.github.com/users/h1alexbel/orgs",
  "repos_url": "https://api.github.com/users/h1alexbel/repos",
  "events_url": "[https://api.github.com/users/h1alexbel/events{/privacy}](https://api.github.com/users/h1alexbel/events%7B/privacy%7D)",
  "received_events_url": "https://api.github.com/users/h1alexbel/received_events",
  "type": "User",
  "site_admin": false,
  "name": "Aliaksei Bialiauski",
  "company": "@Solvdinc",
  "blog": "https://www.h1alexbel.xyz/",
  "location": null,
  "email": null,
  "hireable": null,
  "bio": null,
  "twitter_username": null,
  "public_repos": 72,
  "public_gists": 0,
  "followers": 29,
  "following": 21,
  "created_at": "2021-04-30T11:25:21Z",
  "updated_at": "2024-08-17T08:45:27Z"
}

```

However, we can't add all those fields to our `User` struct. Let's try to wrap it with `ExtUser` that will have ability to find all extra information and print himself as JSON document.

**Comment by 0pdd** [comment]
@h1alexbel 2 puzzles [#141](https://github.com/h1alexbel/fakehub/issues/141), [#142](https://github.com/h1alexbel/fakehub/issues/142) are still not solved.

**Comment by 0pdd** [comment]
@h1alexbel the puzzle [#142](https://github.com/h1alexbel/fakehub/issues/142) is still not solved; solved: [#141](https://github.com/h1alexbel/fakehub/issues/141).

**Comment by 0pdd** [comment]
@h1alexbel all 2 puzzles are solved here: [#141](https://github.com/h1alexbel/fakehub/issues/141), [#142](https://github.com/h1alexbel/fakehub/issues/142).
